README.md

```diff
@@ -445,6 +445,16 @@ dockerBuild {
  * **git**: Adds labels derived from git: current branch, origin repository, latest commit hash and author.
    This label contributor is enabled by default and can be disabled by calling `fromGit(false)`
 
+#### fromProject with variable name
+
+Starting form version 5.2.0 we can use `fromProject(String environmentVariable, Project project)` or `fromProject(String environmentVariable, String projectName)`
```

Comment:
Documentation is just for the last version, so we don't need to specify from which version a function is available.

To look at documentation for older versions, users can just switch to the release tag of that version and read the documentation.